README.md

```diff
@@ -48,33 +63,62 @@ jobs:
 
 ## Advanced usage
 
- ### Custom Certification Authorities or Self-signed certificates
+### Build an image that is not defined in the manifest
 
- You can specify a custom certificate authority or a self-signed certificate by setting the `OKTETO_CA_CERT` environment variable. When this variable is set, the action will install the certificate in the container, and then execute the action. 
+This example sets the context, and then builds an image that is not defined in the Okteto manifest
```

Comment:
@jLopezbarb do you have context on why tag was not supported anymore with 2.0 change?

Reply:
I can't remember. I think it was because with the manifest v2 some flags were skipped so we created v1 and v2 to differentiate between them